### DISCUSSION TITLE
Grafana as data source for Grafana

### DISCUSSION THREAD

**Discussion by knrdl** [grafana/grafana#52692] [Data Sources]
The relevant measurements are written to a public grafana instance (readonly access).
Now I want them to be included into queries/panels within my private grafana instance.
Is this possible??? (Without copying the data)

Please help! Thanks a lot!

**Comment by torkelo** [comment]
no, there is currently no way to query a remote Grafana instance this way

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
